schema.sql

```diff
@@ -38,3 +38,42 @@ ADD CONSTRAINT fk_owners
 FOREIGN KEY(owners_id)
 REFERENCES owners(id)
 ON DELETE CASCADE;
+
+-- Create a table named vets
+CREATE TABLE vets (
+ id SERIAL PRIMARY KEY,
+ name VARCHAR(200),
+ age INT,
+ date_of_graduation DATE
+);
+
+-- Create a joining between species and vet 
+CREATE TABLE specializations (
+  species_id INT NOT NULL,
+  vets_id INT NOT NULL,
+  PRIMARY KEY(species_id, vets_id),
```

Comment:
- [ ] When you make these two Primary Keys, it will not possible to add the same vet or the same species twice. Please drop the PKs.

Reply:
Thank you very much @NduatiKagiri . I have removed the primary key. 👍